src/main/java/org/folio/edge/core/utils/OkapiClient.java

```diff
@@ -52,6 +54,12 @@ public OkapiClient(OkapiClient client) {
     initDefaultHeaders();
   }
 
+  public OkapiClient(OkapiClient client, String secondaryTenantId) {
+    this(client);
+    this.secondaryTenantId = secondaryTenantId;
+    defaultHeaders.set(X_OKAPI_TENANT, StringUtils.isNullOrEmpty(secondaryTenantId) ? tenant : secondaryTenantId);
```

Comment:
I think it might be simpler to do
```
if (!StringUtils.isNullOrEmpty(secondaryTenantId)) {
   defaultHeaders.set(X_OKAPI_TENANT, secondaryTenantId);
 }
 ```
because your code requires you to know that Vertx's MultiMap implementation overwrites the previous key value if both the key and the value are the same. Not every MultiMap implementation does this.

Reply:
done